codechecker_common/plist_parser.py

```diff
@@ -73,8 +73,8 @@ class LXMLPlistParser(plistlib._PlistParser):
     The benefit of this library that this is faster than other libraries so it
     will improve the performance of the plist parsing.
     """
-    def __init__(self, use_builtin_types=True, dict_type=dict):
-        plistlib._PlistParser.__init__(self, use_builtin_types, dict_type)
+    def __init__(self, dict_type=dict):
+        plistlib._PlistParser.__init__(self, dict_type)
```

Comment:
We need to handle old interpreters as well. I recommend to change it to something similar:
```suggestion
    def __init__(self, dict_type=dict):
        # From Python 3.9 the __init__ function is changed:
        # https://github.com/python/cpython/commit/ce81a925ef
        # To be backward compatible with old interpreters we need to call
        # this function based on conditions:
        params = plistlib._PlistParser.__init__.__code__.co_varnames
        if len(params) === 3 and "use_builtin_types" in params:
            # Before 3.9 interpreter.
            plistlib._PlistParser.__init__(self, use_builtin_types, dict_type)
        else:
            plistlib._PlistParser.__init__(self, dict_type)
```

Reply:
Thanks, I applied with with slight modifications.